### PR TITLE
[Circletracer] Add command for circle file and json files visualization by node graph

### PR DESCRIPTION
**Pull request by yesfordev**
This will and command for visualization by node graph.

ONE-vscode-DCO-1.0-Signed-off-by: Youngeun Seo <yesfordev@gmail.com>

### CODE REVIEW

**Comment by yesfordev** [comment]
from draft #179
from issue #66.

File: src/extension.ts [hunk 1]
```diff
@@ -72,8 +72,14 @@ export function activate(context: vscode.ExtensionContext) {
...
+    console.log('one circle tracer...');
+    // TODO PROCESS DIALOG
+  });
+  context.subscriptions.push(disposableOneCircleTracer);
 }
 
 export function deactivate() {
   // TODO do cleanup
-}
+}
```
Comment (llFreetimell) [inline src/extension.ts:85]:
```suggestion
}

```

Please execute `infra/format` to draft in advance :)

File: package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "onCommand:onevscode.json-tracer",
     "onCommand:onevscode.configuration-settings",
     "onCommand:onevscode.toggle-codelens",
-    "onStartupFinished"
+    "onStartupFinished",
+    "onCommand:onevscode.circle-tracer"
```
Comment (llFreetimell) [inline package.json:19]:
```suggestion
    "onCommand:onevscode.circle-tracer",
    "onStartupFinished"
```

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**